include/netlink/route/link/bridge_info.h

```diff
@@ -68,6 +69,19 @@ extern void rtnl_link_bridge_set_mcast_snooping(struct rtnl_link *link,
 extern int rtnl_link_bridge_get_mcast_snooping(struct rtnl_link *link,
 					       uint8_t *value);
 
+extern void rtnl_link_bridge_set_no_linklocal_learn(struct rtnl_link *link,
+						    bool value);
+extern int rtnl_link_bridge_get_no_linklocal_learn(struct rtnl_link *link,
+						   bool *value);
```

Comment:
In this case, we can return the boolean value and an error value at the same time.

In earlier places, where we get a `uint32_t`, a `int` cannot encode a (negative) error value and the return value. So we had the output argument. I think for getters where successful values fit into `[0, INT_MAX]`, we can fit them both together. It's more convenient to use.

Consistency is a concern here, however, we are already not quite consistent (sometimes for good reasons). So, it seems more useful to me to use the API that makes the most sense for the type. Meaning, a getter for a boolean can squeeze the error and the boolean value in an `int`.

Reply:
Not sure I'm sold on the lack of consistency argument but I'll change it anyway